src/en/space-station-14/player-interaction/proposals/night-vision.md

```diff
@@ -0,0 +1,25 @@
+# Night Vision
+
+| Designers | Implemented |
+|---|---|---|
+| pofitlo | :white_check_mark: No |
+
+
+## Overview
+
+adds night vision, which is a dimly lit area around the character that only he sees.
+
+## How it work
+
+using the new client-side system, which will be inherited from the Shared Point Light System, add a very dim light source that only the client sees. 
+for people, this is 1 tile, which only allows you to see yourself on the screen in order to somehow navigate.
+dwarves can see on 2-3 tiles
+lizards by 3-4 tiles
```

Comment:
3 tiles is particularly *huge* for darksight, especially on a roundstart species. It's pretty rare for SS13 codebases to go beyond 2 tiles due to how strong darksight is